src/main/kotlin/no/nav/familie/ef/personhendelse/handler/ForelderBarnHandler.kt

```diff
@@ -0,0 +1,26 @@
+package no.nav.familie.ef.personhendelse.handler
+
+import no.nav.familie.ef.personhendelse.client.SakClient
+import no.nav.familie.ef.personhendelse.client.pdl.PdlClient
+import no.nav.familie.ef.personhendelse.generated.enums.ForelderBarnRelasjonRolle
+import no.nav.familie.ef.personhendelse.util.identerUtenAktørId
+import no.nav.person.pdl.leesah.Personhendelse
+import org.springframework.stereotype.Component
+
+@Component
+class ForelderBarnHandler(val sakClient: SakClient, val pdlClient: PdlClient) : PersonhendelseHandler {
+
+    override val type = PersonhendelseType.FORELDERBARNRELASJON
+
+    override fun lagOppgaveBeskrivelse(personhendelse: Personhendelse): String {
+        return "Bruker har fått et nytt barn"
+    }
+
+    override fun skalOppretteOppgave(personhendelse: Personhendelse): Boolean {
+        val personIdent = personhendelse.identerUtenAktørId().first()
+        val antallBarnPdl =
+            pdlClient.hentPerson(personIdent).forelderBarnRelasjon.filter { it.minRolleForPerson == ForelderBarnRelasjonRolle.BARN }.size
+        val antallBarnSøknad = sakClient.hentFnrForAlleBarn(personIdent).size
+        return antallBarnSøknad != antallBarnPdl
```

Comment:
Er det noe andre regler enn bare å sjekke antall i diff her? Har du hatt dialog med funksjonelle mha terminbarn/annet?

Reply:
Hvilke "barn" er vi interessert i her? Er det kun nyfødte? Isåfall bør vi også sjekke at de "nye" barna ikke er X år gamle, kanskje?